var/spack/repos/builtin/packages/roctracer-dev/package.py

```diff
@@ -76,7 +76,8 @@ def determine_version(cls, lib):
 
     def setup_build_environment(self, build_env):
         spec = self.spec
-        build_env.set("HIP_PATH", spec["hip"].prefix)
+        build_env.set("HIP_PATH", spec["hip"].prefix),
+        build_env.append_flags("CFLAGS", "-fPIC")
```

Comment:
I think the idiomatic way of doing this in CMake is to pass `-DCMAKE_POSITION_INDEPENDENT_CODE=ON` in the cmake_args.

Reply:
@cgmb, Set  CMAKE_POSITION_INDEPENDENT_CODE in cmake_args. Thanks